frontend/src/components/__tests__/LandingView.spec.js

```diff
@@ -0,0 +1,85 @@
+
+    const movieArr = ref([
+        {
+            "name": "The Matrix",
+            "description": "A computer hacker learns from mysterious rebels about the true nature of his reality and his role in the war against its controllers.",
+            "genre": [
+                { "id": 28, "name": "Action" },
+                { "id": 878, "name": "Science Fiction" }
+            ],
+            "duration": 136,
+            "releaseDate": "1999-03-31",
+            "imageUrl": "https://image.tmdb.org/t/p/w500/f89U3ADr1oiB1s9GkdPOEpXUk5H.jpg",
+            "rating": 87
+        },
+        {
+            "name": "Interstellar",
+            "description": "A team of explorers travel through a wormhole in space in an attempt to ensure humanity's survival.",
+            "genre": [
+                { "id": 12, "name": "Adventure" },
+                { "id": 18, "name": "Drama" },
+                { "id": 878, "name": "Science Fiction" }
+            ],
+            "duration": 169,
+            "releaseDate": "2014-11-07",
+            "imageUrl": "https://image.tmdb.org/t/p/w500/rAiYTfKGqDCRIIqo664sY9XZIvQ.jpg",
+            "rating": 81
+        },
+        {
+            "name": "The Dark Knight",
+            "description": "When the menace known as the Joker emerges from his mysterious past, he wreaks havoc and chaos on the people of Gotham.",
+            "genre": [
+                { "id": 28, "name": "Action" },
+                { "id": 80, "name": "Crime" },
+                { "id": 18, "name": "Drama" }
+            ],
+            "duration": 152,
+            "releaseDate": "2008-07-18",
+            "imageUrl": "https://image.tmdb.org/t/p/w500/qJ2tW6WMUDux911r6m7haRef0WH.jpg",
+            "rating": 85
+        }
+    ]);
+
+    /*
+      Load prop data before tests
+    */
+    beforeEach(() => {
+        wrapper = mount(LandingView, {
+            propsData: {
+                listOfMovies: movieArr.value
+            }
+        })
+    })
+
+    test('Load props', () => {
+        expect(LandingView).toBeTruthy();
+        expect(wrapper.vm.listOfMovies).toBeDefined();
+    })
+
+    test('Find first movie from proplist in DOM', () => {
+        expect(wrapper.find('h2').text()).toBe('Movies');
+
+        const firstMovieDescription = wrapper.findAll('p').at(1);
+        const firstMovieTitle = wrapper.findAll('h3').at(0);
+        const firstMovieImgUrl = wrapper.findAll('img').at(0).element.src;
+
+        expect(firstMovieTitle.text()).toBe('The Matrix')
+        expect(firstMovieDescription.text()).toContain('A computer hacker learns from mysterious rebels')
+        expect(firstMovieImgUrl).toBe('https://image.tmdb.org/t/p/w500/f89U3ADr1oiB1s9GkdPOEpXUk5H.jpg')
+
+    })
+
+    test('ArrayOfHighestRated does not contain Interstellar',()=>{
+        expect(wrapper.vm.arrayOfHighestRated.length).toBe(2)
+        expect(wrapper.vm.arrayOfHighestRated.every(movie => movie.name !== 'Interstellar')).toBeTruthy();
+    })
```

Comment:
This test fails for me  - AssertionError: expected 3 to be 2, and AssertionError: expected false to be truthy